R/tokens_group.R

```diff
@@ -0,0 +1,15 @@
+#' recombine documents tokens by groups
+#' 
+#' @param x \link{tokens} object
```

Comment:
this needs 
a) more thorough documentation (which I could supply if you wish)
b) needs to be object-oriented - define a generic then an `@noRd`ed `tokens_group.tokens` method
c) needs to work identical to the dfm method, where it takes docvar names in the same way (since `tokens` objects now carry docvars too.
d) needs examples